client.go

```diff
@@ -162,12 +172,32 @@ var h2Transport = http2.Transport{
 // NewClient creates a RESTful client instance.
 // The instance has a semi-permanent transport TCP connection.
 func NewClient() *Client {
+	return NewClientWInterface("")
+}
+
+// NewClientWInterface creates a RESTful client instance with binded to that interface.
+// The instance has a semi-permanent transport TCP connection.
+func NewClientWInterface(theInterface string) *Client {
```

Comment:
How about renaming theInterface to networkInterface? The term "interface" has a different generic meaning in Go.